test/general/buffer_wrapper.pass.cpp

```diff
@@ -53,7 +53,7 @@ main()
     test(oneapi::dpl::begin(buf), oneapi::dpl::end(buf), data_ptr, size);
     test(oneapi::dpl::begin(buf, sycl::write_only), oneapi::dpl::end(buf, sycl::write_only), data_ptr, size);
     test(oneapi::dpl::begin(buf, sycl::write_only, _ONEDPL_SYCL_NOINIT), oneapi::dpl::end(buf, sycl::write_only, _ONEDPL_SYCL_NOINIT), data_ptr, size);
-#if TEST_NO_INIT_PRESENT
+#if _ONEDPL_NO_INIT_PRESENT
     test(oneapi::dpl::begin(buf, sycl::property::no_init{}), oneapi::dpl::end(buf, sycl::property::no_init{}), data_ptr, size);

```

Comment:
Why we dont use _ONEDPL_SYCL_NOINIT here?

Reply:
When std >= 17, sycl::no_init and sycl::property::no_init{} are available. We want to test both. _ONEDPL_SYCL_NOINIT is an alias to sycl::no_init in that case.